Installation not working with an ament overlay ws

When calling npm install with `$AMENT_PREFIX_PATH = /home/vincentrou/ros2_ws/test_ws/install:/opt/ros/ardent/`
Here is the error : 
```
  CXX(target) Release/obj.target/rclnodejs/src/addon.o
  CXX(target) Release/obj.target/rclnodejs/src/executor.o
../src/executor.cpp:17:32: fatal error: rcl/error_handling.h: No such file or directory
compilation terminated.
rclnodejs.target.mk:118: recipe for target 'Release/obj.target/rclnodejs/src/executor.o' failed
```
The file `build/rclnodejs.target.mk` contains a wrong include path : 
`-I/home/vincentrou/ros2_ws/test_ws/install:/opt/ros/ardent/include`
Instead of : 
`-I/home/vincentrou/ros2_ws/test_ws/install/include`
`-I/opt/ros/ardent/include`

--- a/binding.gyp
+++ b/binding.gyp
@@ -10,9 +10,6 @@
   'targets': [
     {
       'target_name': 'rclnodejs',
-      'variables': {
-        'ROS2_INSTALL_PATH': '<!(echo $AMENT_PREFIX_PATH)',
-      },
       'sources': [
         './src/addon.cpp',
         './src/executor.cpp',
@@ -51,14 +48,14 @@
             'defines': [
               'OS_LINUX'
             ],
-            'include_dirs': [
-              "<(ROS2_INSTALL_PATH)/include/",
-            ],
             'cflags_cc': [
               '-std=c++14'
             ],
-            'libraries': [
-              '-L<(ROS2_INSTALL_PATH)/lib'
+            'include_dirs': [
+              "<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/:/, '/include/ ') + '/include/')\")",
+            ],
+            'library_dirs': [
+              "<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/:/, '/lib/ ') + '/lib/')\")",
             ],
           }
         ],
@@ -68,15 +65,12 @@
             'defines': [
               'OS_WINDOWS'
             ],
-            'variables': {
-              "ROS2_INSTALL_PATH_WINDOWS": "<!(echo %AMENT_PREFIX_PATH%)",
-            },
             'cflags_cc': [
               '-std=c++14'
             ],
             'include_dirs': [
               './src/third_party/dlfcn-win32/',
-              '<(ROS2_INSTALL_PATH_WINDOWS)/include/',
+              "<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/;/, '\\\include ').replace(/\\\/g, '/') + '/include')\")",
             ],
             'sources': [
               './src/third_party/dlfcn-win32/dlfcn.c',
@@ -87,7 +81,7 @@
               },
               'VCLinkerTool': {
                 'AdditionalDependencies': ['psapi.lib'],
-                'AdditionalLibraryDirectories': ['<(ROS2_INSTALL_PATH_WINDOWS)/lib/'],
+                'AdditionalLibraryDirectories': ["<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/;/, '\\\lib ').replace(/\\\/g, '/') + '/lib')\")",],
               }
             }
           }
@@ -99,10 +93,10 @@
               'OS_MACOS'
             ],
             'include_dirs': [
-              "<(ROS2_INSTALL_PATH)/include/",
+              "<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/:/, '/include/ ') + '/include/')\")",
             ],
-            'libraries': [
-              '-L<(ROS2_INSTALL_PATH)/lib'
+            'library_dirs': [
+                "<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/:/, '/lib/ ') + '/lib/')\")",
             ],
             'xcode_settings': {
               'GCC_ENABLE_CPP_EXCEPTIONS': 'YES',

